.github/workflows/ci_tests_dev.yaml

```diff
@@ -1,6 +1,7 @@
 # Tests PyGMT with GMT dev version on Linux/macOS/Windows
```

Comment:
To be consistent with the rest of the documentation and tests (see PR #2578). Similar changes are needed in the files `ci_tests.yaml` and `ci_tests_legacy.yaml`, please.

```suggestion
# Test PyGMT with GMT dev version on Linux/macOS/Windows
```

Reply:
Cool, let me do that tomorrow when I'm back at my computer. Hard to edit multiple files on mobile 😬

Reply:
No hurry, that's only a small change 🙂.

Reply:
Ok, done in 8b203691294e44a4e9ebc5318c06fcacd3e1845b